lib/improver/utilities/save.py

```diff
@@ -142,6 +144,10 @@ def save_netcdf(cubelist, filename):
     Raises:
         warning if cubelist contains cubes of varying dimensions.
     """
+    # To avoid compression SAVE_COMPRESSED=False can be exported to the
+    # environment. Defaults True if not set.
+    save_compressed = ast.literal_eval(os.getenv("SAVE_COMPRESSED", "True"))
```

Comment:
I think I'd prefer a CLI flag really. If we are going to go for an environment variable, it probably should be IMPROVER_SAVE_UNCOMPRESSED and just test it's non-empty. We should also document it in all the CLI help... so it probably doesn't save any effort vs a CLI flag.

Reply:
To me it also feels like reading environment within the save routine doesn't really belong there. Unfortunately, there doesn't seem any other easy way to implement that. CLI flag will force us for us add this to each and every CLI (because we have no way to handle common options across all CLIs).

A nice way to implement this could be with a context manager:

```python
with improver.settings(netcdf_compression=False):
     ...
```

Unfortunately, we have no common point of entry into CLIs where we could add something like this.
CC @arh89, maybe you have some thoughts on this.

Reply:
(Agreed with @bayliffe about not doing this until after @neilCrosswaite's PR has gone in.)

Adding to all CLIs should be trivial, if we add to the `COMPULSORY_ARGUMENTS` in https://github.com/metoppv/improver/blob/master/lib/improver/argparser.py#L87, then all CLIs should pick them up.

Reply:
Also, if we wait until after Neil's work has gone in, we can trivially centralise the loading and saving, then we're golden 👍